src/functions/apis/auth/handler.ts

```diff
@@ -13,15 +13,17 @@ export async function execute(event: ApiGatewayEvent): Promise<APIHttpResponse>
         const request: AuthRequest = Validate(JSON.parse(event.body));
         const connection = await Databases.getConnection();
         const action = new AuthAction(connection);
-        const user = await action.execute(request.username, request.password);
+        const result = await action.execute(request.username, request.password);
```

Comment:
Please change this to 
`const {.user, acceess_token, refresh_token } = await action.execute(request.username, request.password);`